### DISCUSSION TITLE
Question about this repo and android versions

### DISCUSSION THREAD

**Discussion by eternalstarfire** [cfig/Android_boot_image_editor#77] [Q&A]
Hi there
If I want to edit the boot image for my device currently running Android 7, do I need to check out the android-7.1.1_r6 tag and follow the instructions from that tag? Or can I just use the latest main and the latest instructions?

**Comment by cfig** [comment]
For new comers, "master" branch is always recommended.
Legacy tags are for those who only want a working code base that they have verified, and don't need any new features added in later Android versions.
